### PR TITLE
Fixed syntax errors in usage.md

### PR DESCRIPTION
**Pull request by Lord-Protector**
My WebStorm IDE show many errors. With this code all works as should.

### CODE REVIEW

File: doc/usage.md [hunk 1]
```diff
@@ -168,7 +168,7 @@ Every `Error` object we pass back has the properties:
 For example, sending message to a non-existent user:
 
 ```js
-bot.sendMessage(nonExistentUserId, 'text').catch(error => {
```
Comment (GochoMugo) [inline doc/usage.md]: This is correct ES6 syntax.

**Comment by GochoMugo** [comment]
Closing this as it is invalid. Please reopen if you believe otherwise.

We appreciate you taking your time to contribute to this project. :smile:

File: doc/usage.md (hunk 1 above)
Comment (Lord-Protector) [inline doc/usage.md]: @GochoMugo I use Node 6.9.5 and it's doesn't work :(

Comment (GochoMugo) [inline doc/usage.md]: I have just tried it with 6.9.5 to confirm, and it does work. The syntax is supported.

You reported that your IDE is logging the errors. You sure it is using an updated Node.js?

Comment (Lord-Protector) [inline doc/usage.md]: @GochoMugo I think, that my IDE (WebStorm) doesn't understand new syntax. I have just tried and yep, it's working despite errors... Then I pick ES6 it's suggest me to setup Babel Transpiler. So I can just ignore it and use ES6?

Comment (GochoMugo) [inline doc/usage.md]: Well, as long as the end-result is you using ES6, go for it.

(ps: not familiar with the IDE)

**Pull request closed without merging**